<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple FFT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>7</number>
      </property>
      <widget class="QScrollArea" name="scrollAreaSource">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QcvMatWidget" name="sourceImage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>340</width>
          <height>323</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QScrollArea" name="scrollAreaSpectrum">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QcvMatWidget" name="spectrumImage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>339</width>
          <height>323</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidgetFilters">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabImage">
       <attribute name="title">
        <string>Image</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBoxSize">
          <property name="title">
           <string>Size</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QRadioButton" name="radioButtonOrigSize">
               <property name="text">
                <string>Original</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonCustomSize">
               <property name="text">
                <string>Custom</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="labelWidth">
                 <property name="text">
                  <string>Width</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBoxWidth">
                 <property name="maximum">
                  <number>1600</number>
                 </property>
                 <property name="singleStep">
                  <number>4</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="labelHeight">
                 <property name="text">
                  <string>Height</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBoxHeight">
                 <property name="maximum">
                  <number>1200</number>
                 </property>
                 <property name="singleStep">
                  <number>4</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxConv">
          <property name="title">
           <string>Conversions</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkBoxFlip">
             <property name="text">
              <string>Flip</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxGray">
             <property name="text">
              <string>Convert to gray</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxFFT">
          <property name="title">
           <string>FFT</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="labelMag">
               <property name="text">
                <string>Mag.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBoxMag">
               <property name="minimum">
                <number>5</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="labelFFTSizeTitle">
               <property name="text">
                <string>FFT Size</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelFFTSizeValue">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSources">
    <property name="title">
     <string>Sources</string>
    </property>
    <addaction name="actionCamera_0"/>
    <addaction name="actionCamera_1"/>
    <addaction name="actionFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <widget class="QMenu" name="menuSize">
     <property name="title">
      <string>taille</string>
     </property>
     <addaction name="actionOriginalSize"/>
     <addaction name="actionConstrainedSize"/>
    </widget>
    <addaction name="actionFlip"/>
    <addaction name="actionGray"/>
    <addaction name="separator"/>
    <addaction name="menuSize"/>
   </widget>
   <widget class="QMenu" name="menuRender">
    <property name="title">
     <string>Render</string>
    </property>
    <addaction name="actionRenderImage"/>
    <addaction name="actionRenderPixmap"/>
    <addaction name="actionRenderOpenGL"/>
   </widget>
   <addaction name="menuSources"/>
   <addaction name="menuVideo"/>
   <addaction name="menuRender"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCamera_0">
   <property name="text">
    <string>Camera 0</string>
   </property>
   <property name="toolTip">
    <string>caméra interne</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+0</string>
   </property>
  </action>
  <action name="actionCamera_1">
   <property name="text">
    <string>Camera 1</string>
   </property>
   <property name="toolTip">
    <string>caméra externe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+1</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
   <property name="toolTip">
    <string>fichier video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFlip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>flip</string>
   </property>
  </action>
  <action name="actionOriginalSize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Originale</string>
   </property>
   <property name="toolTip">
    <string>taille originale</string>
   </property>
  </action>
  <action name="actionConstrainedSize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
   <property name="iconText">
    <string>contrainte</string>
   </property>
   <property name="toolTip">
    <string>taille imposée</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionRenderImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionRenderPixmap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pixmap</string>
   </property>
  </action>
  <action name="actionRenderOpenGL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenGL</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>gray</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QcvMatWidget</class>
   <extends>QWidget</extends>
   <header>QcvMatWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollAreaSource</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonCustomSize</sender>
   <signal>clicked(bool)</signal>
   <receiver>actionConstrainedSize</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConstrainedSize</sender>
   <signal>triggered(bool)</signal>
   <receiver>radioButtonCustomSize</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonOrigSize</sender>
   <signal>clicked(bool)</signal>
   <receiver>actionOriginalSize</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOriginalSize</sender>
   <signal>triggered(bool)</signal>
   <receiver>radioButtonOrigSize</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFlip</sender>
   <signal>clicked(bool)</signal>
   <receiver>actionFlip</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFlip</sender>
   <signal>triggered(bool)</signal>
   <receiver>checkBoxFlip</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxGray</sender>
   <signal>clicked(bool)</signal>
   <receiver>actionGray</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGray</sender>
   <signal>triggered(bool)</signal>
   <receiver>checkBoxGray</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
